freud/_order.pxd

```diff
@@ -74,6 +74,20 @@ cdef extern from "TransOrderParameter.h" namespace "freud::order":
         unsigned int getNP()
         float getK()
 
+cdef extern from "Steinhardt.h" namespace "freud::order":
+    cdef cppclass Steinhardt:
+        Steinhardt(const freud._box.Box &, float, unsigned int, float,
+                bool, bool, bool) except +
```

Comment:
You'll need to fix the code here to address the `flake8` code style issues that are preventing tests from running. See here for info about the `flake8` pre-commit hook, which can help you catch these early: https://freud.readthedocs.io/en/stable/sourceconventions.html#python